src/main/java/nl/lexemmens/podman/service/SecurityContextService.java

```diff
@@ -67,13 +67,18 @@ private boolean isSELinuxEnabled() throws MojoExecutionException {
          * Memory protection checking:     actual (secure)
          * Max kernel policy version:      33
          */
-        Optional<String> seLinuxStatus = seStatusCommand.execute()
-                .stream()
-                .filter(line -> line.contains("SELinux status"))
-                .map(this::extractSeLinuxStatus)
-                .findFirst();
+        try {
+            Optional<String> seLinuxStatus = seStatusCommand.execute()
+                    .stream()
+                    .filter(line -> line.contains("SELinux status"))
+                    .map(this::extractSeLinuxStatus)
+                    .findFirst();
 
-        return seLinuxStatus.map(seLinuxStatusString -> seLinuxStatusString.equals("enabled")).orElse(false);
+            return seLinuxStatus.map(seLinuxStatusString -> seLinuxStatusString.equals("enabled")).orElse(false);
+        } catch (MojoExecutionException mee) {
+            log.debug(mee.getMessage());
```

Comment:
LGTM! Propose to add a custom debug message like: "SELinux is not installed or not available".
```suggestion
            log.debug("SELinux is not installed or not available", mee.getMessage());
```